packages/build/src/log/messages/core.ts

```diff
@@ -32,10 +33,18 @@ export const logBuildSuccess = function (logs) {
   logMessage(logs, '')
 }
 
-export const logTimer = function (logs, durationNs, timerName, systemLog) {
+export const logTimer = function (logs, prefix, durationNs, timerName, systemLog, outputManager?: OutputManager) {
```

Comment:
This additional parameter lets us print an empty line before the timer, which was the sole purpose of the `logStepSuccess` function we can now remove.